sdk/formrecognizer/Azure.AI.FormRecognizer/src/FieldValue.cs

```diff
@@ -391,5 +397,50 @@ public SelectionMarkState AsSelectionMarkState()
 
             return _fieldValue.ValueSelectionMark.Value;
         }
+
+        /// <summary>
+        /// Gets the value of the field as an ISO 3166-1 alpha-3 country code <see cref="string"/>.
+        /// </summary>
+        /// <returns>The value of the field converted to an ISO 3166-1 alpha-3 country code <see cref="string"/>.</returns>
+        /// <exception cref="InvalidOperationException">Thrown when <see cref="ValueType"/> is not <see cref="FieldValueType.Country"/>.</exception>
+        public string AsCountryCode()
+        {
+            if (ValueType != FieldValueType.Country)
+            {
+                throw new InvalidOperationException($"Cannot get field as country code.  Field value's type is {ValueType}.");
+            }
+
+            if (_fieldValue == null)
+            {
+                return ValueString;
+            }
+
+            return _fieldValue.ValueCountry;
+        }
+
+        /// <summary>
+        /// Gets the value of the field as a <see cref="FieldValueGender"/>.
+        /// </summary>
+        /// <returns>The value of the field converted to a <see cref="FieldValueGender"/>.</returns>
+        /// <exception cref="InvalidOperationException">Thrown when <see cref="ValueType"/> is not <see cref="FieldValueType.Gender"/>.</exception>
+        public FieldValueGender AsGender()
```

Comment:
Does `AsGender` sound good? Maybe `AsFieldValueGender`?
`FieldValueGender` could be renamed as well. `DocumentGender`? `IdDocumentGender`?

Reply:
Looking at the other `AsXX` methods .... maybe `AsGender` is not that bad. 
I also find long `FieldValueGender` and out of place as none of the rest have that prefix.
`DocumentGender` might not imply what we want, and same with `IdDocumentGender` which makes it a constraint to only `IdDocument`.
`ValueGender` is too generic?

Reply:
I don't think that `ValueGender` is too generic, but I also think it's not very descriptive. I saw other languages keeping it as `FieldValueGender`, so I'll do the same for this release. I think we should have a discussion with others about the name (https://github.com/Azure/azure-sdk-for-net/issues/20034).